ext/pdo/pdo_dbh.c

```diff
@@ -517,7 +517,7 @@ PHP_METHOD(PDO, prepare)
 		}
 		if ((item = zend_hash_index_find(Z_ARRVAL_P(value), 0)) == NULL) {
 			zend_value_error("PDO::ATTR_STATEMENT_CLASS value must be an array with the format "
-				"array(classname, array(ctor_args))");
+				"array(classname, array(constructor_args))");
```

Comment:
```suggestion
				"array(classname, constructor_args)");
```
Would be better, I think?

Reply:
Yes, the current one is potentially misleading